binary_gentoo/internal/cli/build.py

```diff
@@ -125,6 +126,22 @@ def parse_command_line(argv):
                         action='store_true',
                         help='enforce installation (default: build but do not install)')
 
+    parser.add_argument('--use',
+                        dest='use',
+                        default='',
+                        help='custom one-off use flags (default: none)')
+
+    parser.add_argument('--update',
+                        dest='update',
+                        default=False,
+                        action='store_true',
+                        help='do an emerge --update (default: False)')
+
+    parser.add_argument('--tag-docker-image',
+                        dest='tag_docker_image',
+                        default='',
+                        help='create an image of the resulting container')
```

Comment:
My vote for:
```diff
--- an image
+++ a Docker image
```
Maybe same for container(?).

Reply:
Agree with the precision to add "Docker image", but I'd say it's rather obvious we are talking about Docker containers so I wouldn't mention it twice.

Reply:
Sounds good to me